### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by bronzdoc**
Adds missing issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+**Before we start...:**
+
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+- [ ] I'm reporting the issue to the correct repository (for multi-repository projects)
+
+
+**Version, Branch, or Commit:**
+
+Inform what version, branch, commit of Skunk you are using.
```
Comment (arielj) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]: it says Skunk here

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+**Before we start...:**
+
+- [ ] I checked the documentation and didn't find this feature
+- [ ] I checked to make sure that this feature has not already been requested
+
+
+**Branch/Commit:**
+
+Inform what branch/commit/version of "Skunk" you are using.
```
Comment (arielj) [inline .github/ISSUE_TEMPLATE/feature_request.md:20]: same here

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+3. Operating System version
+4. Ruby version
+
+_Delete any information that is not relevant._
+
+**Logs**
+
+Include relevant log snippets or files here.
+
+**I will abide by the [code of conduct] (CODE_OF_CONDUCT.md)**
```
Comment (arielj) [inline .github/ISSUE_TEMPLATE/bug_report.md:53]: this is linking to a file that we don't have, should we add it too in this PR?

**Review by arielj — CHANGES_REQUESTED** [review]
hey @bronzdoc just added some comments

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (bronzdoc) [reply .github/ISSUE_TEMPLATE/bug_report.md:21]: Do you think we should add something more general like "the project" instead?

Comment (arielj) [reply .github/ISSUE_TEMPLATE/bug_report.md:21]: yes, that should make it easier to reuse 👍

**Comment by etagwerker** [comment]
@bronzdoc Ping

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (bronzdoc) [reply .github/ISSUE_TEMPLATE/bug_report.md:53]: This should be fixed after https://github.com/fastruby/rails-template/pull/11 is merged

**Review by arielj — APPROVED** [review]
LGTM!

**Pull request merged**